src/Modal/ModalContent.tsx

```diff
@@ -29,10 +29,10 @@ const ModalContent = ({ onClose, onSave, initialDate }: ModalContentProps) => {
 
   const handleSubmit = (e: React.FormEvent) => {
     e.preventDefault();
-
-    const startDateTime = new Date(`${startDate}T${startTime}`);
-    const endDateTime = new Date(`${endDate}T${endTime}`);
-
+    const startDateTime = new Date(`${startDate}T${startTime}:00`).toString();
+    const endDateTime = new Date(`${endDate}T${endTime}:00`).toString();
+    
+    
```

Comment:
maybe built in methods could be used?
eg. [toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)?

Now it looks like complex logic for rather simple thing, because strings are concatenated, converted to date and then converted again to string.

Reply:
Changed this, but not much of a difference since I have to merge startDate and startTime anyways.